pycytominer/cyto_utils/cells.py

```diff
@@ -714,7 +728,9 @@ def merge_single_cells(
                 ]
 
                 if isinstance(sc_df, str):
-                    sc_df = self.load_compartment(compartment=left_compartment)
+                    sc_df = self.load_compartment(
+                        compartment=left_compartment, features=self.features
```

Comment:
should this be `features=features` instead of `features=self.features`?

Otherwise this doesn't seem like your changes have an effect in `merge_single_cells` - is this intended?

Reply:
The same comment goes for the changes added below in this function as well.

Reply:
My assumption was that `features` was intended to be passed when defining `SingleCells` (ie `database = SingleCells(f"sqlite:///{backend_file}",features=desired_features)`; in that case, yes, merge_single_cells DOES pick up these changes). 

Do we ever expect/want the user to change what the desired features are AFTER instantiating the class object? I was thinking no, but you tell me.

Reply:
> Do we ever expect/want the user to change what the desired features are AFTER instantiating the class object?

Hmm, I'm still a bit confused. What is your entry point to allow column subsetting?

If the entry point is upon class instantiation, then why do you need to add a `features` argument to `load_compartment()` ([here](https://github.com/cytomining/pycytominer/pull/246/files#diff-ed9097fc5b9747e8287afce4ba63d18724e3e88d7725fabda9b28be2e6ab81f2R452))?

(I agree that a user shouldn't need to change the features arg after instantiation)